BenGWeeks/Zapp.ie
# Bug 97121: Bot notification message is duplicating

### Description

The logic that triggered the error, 'FYI I wasn't able to message Nataliia Borovyk that they have a zap from you because they don't have me installed yet - maybe you could ping them and let them know to install Zapp.ie!' has been removed. Notifying the user that new Sats have been sent to them will now occur through a separate channel. (this logic is being developed in this ticket - [Enhancement 94801](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/94801): Zappie - Recipient of Sats not receiving notifications)

### Screenshot/video

![image](https://github.com/user-attachments/assets/aa53265d-aee4-4793-98a6-92552e2c6765)

### Related workitems

- [Bug 97121](https://dev.azure.com/EvrosPowerPlatformTeam/Default/_workitems/edit/97121): Bot notification message is duplicating


### Type of Change


### Test Instructions

To test this PR:

1. Open the Zapp.ie (local) bot.
2. Select the "Send Zap" prompt.
3. Fill out the form and submit the request.
4. Verify that the notification error, "FYI I wasn't able to message Nataliia Borovyk that they have a zap from you because they don't have me installed yet - maybe you could ping them and let them know to install Zapp.ie!" no longer appears.

### Checklist:


fyi @EdiWeeks
